bananabot.py

```diff
@@ -128,25 +163,26 @@ def about():
 @auth.login_required
 def admin():
     if request.method == 'POST':
-        formId = request.form['formId']
-        app.logger.debug("Form ID: " + str(formId))
-        if formId == 'bananaTime':
-            setBananaTime(request.form['bananaTime'])
-        elif formId == 'morningAnnouncementTime':
-            setMorningAnnouncementTime(request.form['morningAnnouncementTime'])
-        elif formId == 'announcement':
-            addMinsBeforeAnnouncement(request.form['minsBefore'], request.form['message'])
-        elif formId == 'deleteAnnouncement':
-            removeMinsBeforeAnnouncement(request.form['minsBefore'])
-        elif formId == 'status':
-            toggleStatus()
+        form_id = request.form['form_id']
+        app.logger.debug("Form ID: " + str(form_id))
+
+        if form_id == 'status':
+            toggle_status()
+        elif form_id == 'set_banana_time':
+            set_banana_time(request.form['banana_time'])
+        elif form_id == 'set_banana_time_text':
+            set_banana_time_text(request.form['text'])
+        elif form_id == 'remove_announcement':
+            remove_announcement(request.form['announcement_id'])
+        elif form_id == 'add_announcement':
+            add_announcement(request.form['time'], request.form['text'])
+        elif form_id == 'add_mins_before_announcement':
+            add_mins_before_announcement(request.form['mins_before'], request.form['text'])
 
         return redirect(url_for('admin'))
```

Comment:
Is this supposed to return a HTTP status code to acknowledge the operation? also it seems like no error log or invalid request code is returned in the case that the `form_id` is invalid.